### PR TITLE
provide a quick script to free the pool

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

File: scripts/mkcloudhost/mkcloud_reserve_pool [hunk 1]
```diff
@@ -3,7 +3,14 @@
...
 delay=$3
-message="Did you free VM ${pool_vm} on mkch${host}.cloud.suse.de ?"
+message=$(cat <<EOF
+Did you free VM ${pool_vm} on mkch${host}.cloud.suse.de ?
+
+to free the pool again call:
+
+$(dirname $0)/mkcloud_free_pool ${host} ${pool_vm}
+EOF
+)
```
Comment (jdsn) [inline scripts/mkcloudhost/mkcloud_reserve_pool]: Why so complicated? This should do as well:
```
+ message="Did you free VM ${pool_vm} on mkch${host}.cloud.suse.de ?
+
+to free the pool again call:
+
+$(dirname $0)/mkcloud_free_pool ${host} ${pool_vm}
+"
```

**Comment by jdsn** [comment]
:+1: for either way :)

**Comment by nicolasbock** [comment]
+1

**Comment by jdsn** [comment]
rebase pending, do **not** merge yet

**Pull request merged**